R/support_panel.R

```diff
@@ -71,8 +72,10 @@ support_panel <- function(
 
   if (is_valid_repo_name(repo_name) == FALSE) {
     stop(
-      "Please ensure the repo_name argument points to a repository on the
-      dfe-analytical-services GitHub area."
+      "Please ensure the repo_name argument is a valid URl to a repository on
```

Comment:
think this is a lower case l here and not upper case?

Reply:
Excellent spot! Have fixed that and pushed a commit up for it now